server/stores_test.go

```diff
@@ -0,0 +1,137 @@
+package server
```

Comment:
1. were there no tests here before?
1. these tests look like they need to be fleshed out for at least a couple of non-superadmin roles attempting to GET sources and getting only the ones for their role

Reply:
1. There were no tests here before. We should add tests for the rest of the resources. I didn't add them for the other ones here because it didn't feel appropriate. I opened https://github.com/influxdata/chronograf/issues/2247

2. Added more tests. I could have sworn I did this previously but I guess I didn't

Reply:
these tests are really just checking to see that values are set on context. I'll fix the title.

Reply:
Good catch on this. I was fooled because I had testcoverage

Reply:
👍